resources/migrations/000_migrations.yaml

```diff
@@ -3581,3 +3581,16 @@ databaseChangeLog:
                   defaultValueBoolean: false
                   constraints:
                     nullable: false
+  - changeSet:
+      id: 55
```

Comment:
this will have to be changed to id: 56 (and moved to the bottom of the file) since we recently added a migration 55